scripts/hf_sim.py

```diff
@@ -454,3 +478,5 @@ def validate_end(idx_n):
             validate_end(work_idx[c0 - 1] + 1)
 os.remove(in_stats)
 print("Process %03d of %03d finished (%.2fs)." % (rank, size, MPI.Wtime() - t0))
+if is_master:
+    logger.debug("Simulation completed")
```

Comment:
"." for consistency with other statement messages.

Reply:
Done